### PR TITLE
Master

### PR DESCRIPTION
**Pull request by nbl1268**
Show Program name in side panel

### CODE REVIEW

File: Forms/Main.cs [hunk 1]
```diff
@@ -826,7 +827,7 @@ private void ShowDownloadInfo(int epid)
             }
 
             this.SetToolbarButtons(buttons.ToArray());
-            this.SetSideBar(TextUtils.StripDateFromName(info.Name, info.Date), infoText, Model.Episode.GetImage(epid));
+            this.SetSideBar(TextUtils.StripDateFromName(progInfo.Name + " - " + info.Name, info.Date), infoText, Model.Episode.GetImage(epid));
```
Comment (ribbons) [inline Forms/Main.cs]: The programme name shouldn't be passed to `StripDateFromName`as programme names don't tend to contain dates.  I'd also personally favour a colon and a space as the separator.

File: .gitignore [hunk 2]
```diff
@@ -14,3 +14,4 @@ packages
 .directory
 Thumbs.db
 *.userprefs
+*.ffs_db
```
Comment (ribbons) [inline .gitignore]: Could you add this in a separate commit (doesn't need to be a separate PR though) & say in the commit message what application generates it?

**Comment by ribbons** [comment]
#59 may also be relevant as we are making changes in that area.

**Comment by nbl1268** [comment]
Thanks. I'm going to close this PR, and get my fork to rebase properly.
Then do a separate commit for .gitignore.
Will have a look at #59 and see what i can work out with that and adding the programme name to the sidebar.

**Comment by ribbons** [comment]
Just a heads up - there is no need to open a new PR, just reopen this PR, force push after your rebase and it will update automatically.

**Comment by nbl1268** [comment]
Ok thanks. Only had a quick look at #59 but thinking that the changes that might be needed will be much more than i had initially put in this PR so a new PR for that might be a better thing to do.

Havent been able to sort out rebasing my fork against your master such as to remove those unnecessary commits.
When i use the `git rebase ribbons/master` in my git shell i end up with errors and patch file that i dont know what to do with, and as such have created a whole bunch more commits in my master that i also dont know how to remove. 
Considered deleting my Master repository but that seemed a bit too rash.

**Comment by ribbons** [comment]
I'd suggest doing something like:

```
git branch old-master
git reset --hard ribbons/master
```

That will save your old commits as `old-master` in case you need to reuse anything and then reset your master branch to the same state as my master branch.

**Comment by nbl1268** [comment]
thanks, those commands followed by this one did the trick
`git push -f origin master`
